scripts/ci/build_and_test.sh

```diff
@@ -17,17 +17,14 @@ SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
 source ${SCRIPT_DIR}/parse_args.sh "$@"
 
-if [[ -f ~/.cache/proxy_config ]]; then
```

Comment:
Why remove it? It's done in `build_and_test.sh` instead?

Reply:
This is not necessary for build servers in `CN-HK` regions, also it will cause bazel download fail for arm server in beijing or shanghai region

Reply:
We need this for AARCH64 CI since it could not use CN-HK.